src/node-fallbacks/crypto.js

```diff
@@ -79,3 +80,13 @@ if (timingSafeEqual) {
 }
 
 export const webcrypto = crypto;
+
+export default {
+  ...cryptoBrowserify,
+  getRandomValues,
+  randomUUID,
+  timingSafeEqual,
```

Comment:
Some of these don't make much sense anymore, but we can address that in a follow-up PR

This was originally used in bun's runtime as well. Until we started added more stuff to it.